containers/EmpSponsors.ts

```diff
@@ -120,9 +120,7 @@ const useEmpSponsors = () => {
           let newPositions: SponsorMap = {};
           let newLiquidations: LiquidationMap = {};
 
-          const collReqFromWei = parseFloat(
```

Comment:
Wait, but the collateral can be non-18 decimal, right? So wouldn't we _want_ to pass in `collDec` to make sure the `fromWei` call uses the correct number of decimals for the conversion? I feel like I'm probably just not seeing something obvious :)

Reply:
I was just following the tests for the EMP which uses 18 decimals to set contract-state variables. I think all contract state values are assumed to be in 18 decimals: https://github.com/UMAprotocol/protocol/blob/master/packages/core/test/financial-templates/Liquidatable.js#L1669

Reply:
The collateral requirement is only used in Liquidatable's `_settle` method (please correct me if I'm wrong), and it is used to multiply by `tokensOutstanding` to derive the `tokenRedemptionValue`. So I think it actually has to have same precision as `tokensOutstanding`: https://github.com/UMAprotocol/protocol/blob/master/packages/core/contracts/financial-templates/expiring-multiparty/Liquidatable.sol#L526

Reply:
Ahhh, sorry for the confusion. You are 100% right. I _was_ missing something obvious haha. This is the collateral _requirement_, which is a percentage (I should have read more carefully). As you said, these sorts of numbers always have a precision of 18 decimals in the contract, since the fixed point library understands 1.2e18 to mean 1.2 (or 120%). Once again, sorry for the confusion. LGTM!